### ISSUE TITLE
Can't theme pagination actions toolbar.

### ISSUE THREAD

**Issue by WiXSL** [marmelab/react-admin#4008]
The `Pagination` component uses a `PaginationActions` component which renders the numbers/prev/next  items and is the only way to pass css classes, but is inaccessible outside of the `Pagination` component and the classes used there are hardcoded. 

The documentation suggests coding another component for the case of wanting a “<previous - next>” like toolbar but nothing that allows to style what is being render.

```jsx
...

const Pagination = ({
    loading,
    page,
    perPage,
    rowsPerPageOptions,
    total,
    setPage,
    setPerPage,
    ...rest
}) => {

...

return (
        <TablePagination
            ...
            ActionsComponent={PaginationActions}
            ...
        />
    );
...
```

**Comment by fzaninotto** [comment]
Sorry, I have trouble understanding this issue. Can you please explain, not why the current solution isn't adapted to your need, but what your need actually is?

**Comment by WiXSL** [comment]
Sorry, 
I need to style the item in the pagination toolbar buttons of a List component. (Next 1, 2, 3... Prev>)

**Comment by fzaninotto** [comment]
You need to style them... like, change the color, or change the arrangement / number of buttons? Also, why isn't writing a custom Pagination component a good solution?

**Comment by WiXSL** [comment]
Because is to match code to override if i want all the functionality of `Pagination` and `PaginationActions` components.

**Comment by fzaninotto** [comment]
What would you suggest?

**Comment by WiXSL** [comment]
Maybe add a `classes` prop to `Pagination` component so it can bee passed to `PaginationActions` at least.

Or make the `ActionsComponent` prop of `TablePagination` a prop of `Pagination` component as well to override just the `PaginationCoomponent`

**Comment by fzaninotto** [comment]
fixed by #4132
